src/neptune/JobManagerParallel.py

```diff
@@ -478,7 +478,7 @@ def createDatabaseJob(
 
             for line in inputFile:
 
-                if line[0] is ">":
```

Comment:
This is a nitpick, so feel free to ignore it. But you can use the `startswith` method now to test for a string starting with one or more characters e.g. it could be re-written as `line.startswith('>')`

Reply:
Thanks, this is a good suggestion!

Reply:
Updated in 32838e1